api/jobs.go

```diff
@@ -531,7 +531,7 @@ func (j *Jobs) Dispatch(jobID string, meta map[string]string,
 // Revert is used to revert the given job to the passed version. If
 // enforceVersion is set, the job is only reverted if the current version is at
 // the passed version.
-func (j *Jobs) Revert(jobID string, version uint64, enforcePriorVersion *uint64,
+func (j *Jobs) Revert(jobID string, version *uint64, versionTag *string, enforcePriorVersion *uint64,
```

Comment:
version becomes optional, and therefore now a pointer.

Reply:
We can't change the signature of `api` package methods. We'll need a new `Revert`-like method to do this.

Better yet, we can do what we did on the Tag/Untag Versions PR and query for the version for a given tag in the CLI/UI and then leave this unchanged.